dangerzone/gui/main_window.py

```diff
@@ -486,7 +486,7 @@ def start_button_clicked(self):
 
 
 class ConvertThread(QtCore.QThread):
-    finished = QtCore.Signal(bool)
+    is_finished = QtCore.Signal(bool)
```

Comment:
This fixed a TypeError, `line 492, in run self.finished.emit(self.error) TypeError: finished() only accepts 0 argument(s), 1 given!`

But after a second look, I'm not sure about this solution. I believe the TypeError was caused by Qt 6 [defining `void QThread::finished()`](https://doc.qt.io/qt-6.2/qthread.html#finished) as `[private signal]`, whereas Qt 5 had [defined it](https://doc.qt.io/qt-5/qthread.html#finished) as a `[signal]`. PySide6 raises an exception when we try to manually `emit` that private signal.

Reply:
A more elegant solution could come from avoiding subclassing QThread at all.

Reply:
This might actually have been a bug introduced in https://github.com/freedomofpress/dangerzone/commit/ea47a2e92ce957f0f9e104290bc433f047dddd99. Among other things, this commit adds a bool argument to the `finished` signal. However this didn't make much sense because [the slot received zero arguments](https://github.com/freedomofpress/dangerzone/commit/ea47a2e92ce957f0f9e104290bc433f047dddd99#diff-cca675cf576ebaa4fde44bfc6790124ccc293f2edcbae1d4068c347cc8a83102R540).

It sounds like when the argument became private (which sounds weird in python lingo, where nothing is really private) it now started complaining. So in a way, this was a bug that just didn't show itself before.

Reply:
Resolved, see 5af3d73.

Reply:
resolved via https://github.com/freedomofpress/dangerzone/commit/5af3d7396e4004ce83272710db615fbd1be446a7